docker-compose.yaml

```diff
@@ -34,6 +35,8 @@ services:
       - RUST_BACKTRACE=1
       - CHAINLINK_DEV=true
       - EXPLORER_URL=ws://172.16.1.103:8080
+      - EXPLORER_ACCESS_KEY=u4HULe0pj5xPyuvv
```

Comment:
Would be nice to note somewhere that this is intended for development. We already get quite a few false positives for bug reports about having credentials in the repo. Maybe a different filename or leave a comment?